docs/about/features.rst

```diff
@@ -1,7 +1,3 @@
-.. # Copyright (C) 2020-2024 Intel Corporation

```

Comment:
NIT: In some places, we are removing this header, while in others, it is still present. Can we standardize it one way or the other?

Reply:
I removed headers in files I touched; no reason why not to remove for all docs...
At times it is either out of date, or not in the format as consistent across the framework. An elegant approach is to have a pre-commit check that applies the correct template on all files (and warns if missing or incorrect). 

Could you check if we can integrate a pre-commit tool that updates all headers?

Reply:
Could perhaps check pre-commit hooks such as [this](https://github.com/Quantco/pre-commit-mirrors-insert-license) one.

Reply:
This may work for updating headers, but won't check (?).
I came across this [go based tool](https://github.com/google/addlicense). Will it be easy to integrate in CI, or as pre-commit check?